### PR TITLE
V1 - Remove 'build-info published' log message

### PR DESCRIPTION
**Pull request by eyalbe4**
- [x] All [tests](https://github.com/jfrog/jfrog-client-go#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
This is part of a fix for V1, which has already been implemented for V2 a few months ago.
This PR removes the "Build info successfully published" log message. It is a duplicate of https://github.com/jfrog/jfrog-client-go/pull/419 which has been merged to v2 a few months ago. This was done, in order to support a fix for an issue, whereby the build-info link displayed in the log, was incorrect, when the build is published as part of a project. 
As part of the fix, the fixed link is built and displayed by jfrog-cli-core, and is therefore removed from jfrog-client-go in this PR.
A PR for jfrog-cli-core will be soon created.

### CODE REVIEW

File: auth/cert/sslutils_default.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```
Comment (RobiNino) [inline auth/cert/sslutils_default.go:1]: Is this relevant to the PR?

File: auth/cert/sslutils_windows.go [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//go:build windows
```
Comment (RobiNino) [inline auth/cert/sslutils_windows.go:1]: Is this relevant to the PR?

**Review by RobiNino — CHANGES_REQUESTED** [review]

File: auth/cert/sslutils_default.go (hunk 1 above)
Comment (eyalbe4) [reply auth/cert/sslutils_default.go:1]: It is not, but this is something go 1.17 automatically modifies. It did the same for v2 as well, after we started releasing it with go 1.17.
I'll make sure that v1 is also moved to go 1.17.

File: auth/cert/sslutils_windows.go (hunk 2 above)
Comment (eyalbe4) [reply auth/cert/sslutils_windows.go:1]: It is not, but this is something go 1.17 automatically modifies. It did the same for v2 as well, after we started releasing it with go 1.17.
I'll make sure that v1 is also moved to go 1.17.
